dewy/chunk/router.py

```diff
@@ -21,16 +21,18 @@ async def list_chunks(
     document_id: Annotated[
         int | None, Query(description="Limit to chunks associated with this document")
     ] = None,
-    page: int | None = 0,
+    page: Annotated[
+        int | None, Query(description="Page number to fetch. The first page of results is page 1")
```

Comment:
Why start at 1? 0 indexing is the way?

Reply:
Suggest maybe we move away from this -- instead just use a "page token" -- you give it an optional "token" and the count, and it gives you the next page and the next page token. Then we can choose how to use the token -- we can make it an offset, etc.